lib/job/arGenerateFindingAidJob.class.php

```diff
@@ -24,102 +24,99 @@
 
 class arGenerateFindingAidJob extends arBaseJob
 {
+  // Required parameters:
+  // - Parameters that are mandatory for the job
+  // - They are checked at the begining of the job from arBaseJob run() function
+  // - 'id' and 'name' are required parameters for all jobs and they are added in arBaseJob
+  // - The job will fail if any of the required paramaters are missing
+  protected $requiredParameters = array('objectId');
```

Comment:
This comment repeated each time you redefine the member is a bit too verbose. You could just use the phpdoc tag @see, e.g. ```@see arBaseJob::$requiredParameters```.